src/views/settings/Main.vue

```diff
@@ -748,7 +739,7 @@ export default defineComponent({
       store.commit('setBatterySystemMode', batterySystemModeRef.value);
       switch (batterySystemModeRef.value) {
         case 'real':
-          batterySystemPointCRef.value = 0xE7;
+          batterySystemPointCRef.value = 0xD2;
```

Comment:
@justdecodeme  The battery system logic change was not requested. Could you please revert it to its original state?

Reply:
@ercmz Fixed.